### DISCUSSION TITLE
Disabling navigation

### DISCUSSION THREAD

**Discussion by Sp3EdeR** [Mr-MIBonk/M.I.B._More-Incredible-Bash#266] [General]
Hello everyone,

I enabled navigation before I checked whether the unit has GPS attached. I wonder if there's a simple way to re-disable navigation while leaving other tweaks as-is? Or can it only be done by restoring all raw data sets and then redoing the tweaks?

(I would of course be up for installing a GPS receiver, but there is a plastic cone over the head unit screw that I can't seem to remove without risk of breaking it... So while there's no GPS receiver, it would be better to disable the function, I guess.)

**Comment by harman-f** [comment]
Share a picture of that screw

GPS will not be damaged by enabling it.

**Reply by Sp3EdeR** [reply]
I don't have it apart right now, but I seem to be unable to remove these plastic tabs without damaging them (from a random youtube video):
![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/22780322/0a15bf36-28db-4092-b73e-535f203261dc)
